packages/config/config/devices/0x016A/OOMIDW1.json

```
@@ -0,0 +1,55 @@
+// Omni by Fantem OOMIDW1 and FT112-A
+// Door/Window Sensor
+{
+	"manufacturer": "Fantem",
+	"manufacturerId": "0x016A",
+	"label": "OOMIDW1",
+	"description": "Door/Window Sensor",
+	"devices": [
+		{
+			"productType": "0x0202",
+			"productId": "0x004a"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		}
+	},
+	"paramInformation": {
+		"1": {
+			"$import": "../0x0086/templates/aeotec_template.json#invert_state_report"
```

Just to avoid confusion, a short comment explaining why you import from Aeotec templates would be good. E.g.:
```suggestion
		// Fantem is an OEM for Aeotec devices
		"1": {
			"$import": "../0x0086/templates/aeotec_template.json#invert_state_report"
```

---

Looks good otherwise

---

Great thanks.  I will do similar for the second device in the issues log and submit.  I wanted to get this one approved before moving on.